Sources/PackageManagerDocs/Documentation.docc/IntroducingPackages.md

```diff
@@ -4,8 +4,49 @@ Learn to create and use a Swift package.
 
 ## Overview
 
-overview content here....
+A package consists of a `Package.swift` manifest file along with source files, resources, and other assets. 
+The manifest file, or package manifest, defines the package's name and its contents using the PackageDescription module.
 
-### First Section
+Each package declares `Products`, a list of what the package produces.
+Types of products include libraries, executables, and plugins:
 
-First section content
+- A library defines a module that can be imported by other code. 
+- An executable is a program that can be run by the operating system.
+- A plugin is executable code that the Swift Package Manager may use to provide additional commands or build capabilities.
+
+The package can also declare `Dependencies`, most frequently other Swift packages that provide modules you use.
+Dependencies can also provide macros, plugins, or reference system or binary (non-source) dependencies.
+
+Each product is made up of one or more `Targets`, the basic building block of a Swift package.
+Each target specifies an output, may declare one or more dependencies on other targets within the same package and on products vended by the package’s dependencies.
+A target may define a library, a test suite, an executable, an macro, a binary dependency, and so on.
+
+### About Modules
+
+A Swift package organizes code into _modules_.
+A module specifies a namespace and enforces access controls on which parts of the code can be used outside of that module.
+When you import a library in Swift, you're importing a module to use from your code, regardless of what language was used to create that module.
```

Comment:
I think we should be careful with the terminology here. Only a module can be imported and a library can vend one or more modules.

Reply:
I think this most recent update brings it together, but I'm uncertain if I'm incorrectly conflating module and target, and should be preferring to describe one over the other in terms of introducing the core concepts for a Swift package. Please let me know what you think.